qrest/conf.py

```diff
@@ -223,6 +223,7 @@ def __init__(
         processor: Optional[Type[Resource]] = None,
         description: Optional[str] = None,
         path_description: Optional[dict] = None,
+        timeout: Optional[tuple] = (0, 0)
```

Comment:
This value isn't optional anymore:
``` python
timeout: Tuple[int, int] = (0, 0)
```

Reply:
However, please have a look at my comment on `apply_default_timeout`. It might be that we have to set the initial timeout to None to indicate no timeout has been set.

Reply:
Indeed, from a _user perspective_ it is optional, but it's not optional _from a Python perspective_: it always has a tuple as a value. It would be optional from a Python perspective if it can also hold the value `None` but that's not the case.

One remark about the current type hints in qrest, they're never actually checked and there will be a lot of incorrect ones. Fixing this is the goal of issue #17 , which I want to tackle next sprint.

Reply:
Right, I understand. Fixed it.